### PR TITLE
app-portage/emlop: add 0.7.0

### PR DESCRIPTION
**Pull request by vincentdephily**
* New [upstream major version](https://github.com/vincentdephily/emlop/releases/tag/0.7.0)
* Bumped crate deps and MSRV
* New `emlop.toml` file installed in docs
* Fixes https://bugs.gentoo.org/922533

@Flowdalic 

Thank you.

### CODE REVIEW

File: app-portage/emlop/emlop-0.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Auto-Generated by cargo-ebuild 0.5.4
+
+EAPI=8
+
+CRATES="
+	adler-1.0.2
```
Comment (thesamesam) [inline app-portage/emlop/emlop-0.7.0.ebuild:9]: Please run `pkgcheck scan` and address any warnings.

File: app-portage/emlop/emlop-0.7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,124 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Auto-Generated by cargo-ebuild 0.5.4
```
Comment (thesamesam) [inline app-portage/emlop/emlop-0.7.0.ebuild:4]: We recommend using pycargoebuild at this point (as it won't generate output causing warnings, for a start).

File: app-portage/emlop/emlop-0.7.0.ebuild (hunk 1 above)
Comment (vincentdephily) [reply app-portage/emlop/emlop-0.7.0.ebuild]: Done.

File: app-portage/emlop/emlop-0.7.0.ebuild (hunk 2 above)
Comment (vincentdephily) [reply app-portage/emlop/emlop-0.7.0.ebuild]: Looks like pycargoebuild doesn't generate

    # rust does not use *FLAGS from make.conf, silence portage warning                                                                                                                                                                          
    QA_FLAGS_IGNORED="usr/bin/${PN}"

TBH I don't fully understand what it does, so I left it in for now. Should I remove it ?

Comment (Flowdalic) [reply app-portage/emlop/emlop-0.7.0.ebuild]: Creating the right value for QA_FLAGS_IGNORED is non-trival for tools. And it is still required to silence the QA warning, so we want to keep it.

**Pull request closed without merging**